lib/tapioca/compilers/dsl/url_helpers.rb

```diff
@@ -0,0 +1,82 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+
+begin
+  require "rails"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      class UrlHelpers < Base
+        extend T::Sig
+
+        sig { override.params(root: Parlour::RbiGenerator::Namespace, constant: T.class_of(Object)).void }
```

Comment:
I figured it was better to use `T.class_of(Object)` than `T.untyped`.

Reply:
```suggestion
        sig { override.params(root: Parlour::RbiGenerator::Namespace, constant: T.class_of(Module)).void }
```
would be better, since they are all modules.